src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -83,6 +89,9 @@ class SidebarScreen extends Component {
      * Selecting an item on CreateMenu or closing it by clicking outside of the modal component
      */
     hideCreateMenu() {
+        if (window.document) {
+            window.document.removeEventListener('dragover', this.dragOverListener);
```

Comment:
We need to refactor this component to have web platform-specific changes in the `index.web.js`.

Please check the the guidelines here: https://github.com/Expensify/App#platform-specific-file-extensions

Reply:
Thank you for your comment @mananjadhav . 
When I saw that it didn't work on android and ios, I wanted to do this too. Frankly, I was afraid to repeat too much code for 10 lines. If code duplication is not a problem, I would like to do it as follows, if you see fit. 
I'm a little afraid that the same codes will repeat. I wanted to get your permission before I did it.

![WhatsApp Image 2022-05-15 at 22 22 59](https://user-images.githubusercontent.com/6093207/168490587-67ae6f0c-4e30-4e19-b9ad-b51229a6406c.jpeg)

Reply:
In these cases we should push all the common code to a component say `BaseSidebarScreen`. We then add/override only platform-specific functions and noops in others.

A quick small example, would look like this:

```
BaseSidebarScreen.js

const propTypes = {
      beforeHideCreateMenu: PropTypes.func,
}

class BaseSidebarScreen {

    hideCreateMenu() {
       if(this.props.beforeHideCreateMenu) { 
           this.props.beforeHideCreateMenu();
      }. 

       this.setState({
            isCreateMenuActive: false,
        });
    }
   

}

index.web.js
class SidebarScreen {   
    render() {
       return <BaseSidebarScreen ... />
    }
}


`index.js`
class SidebarScreen {   
    render() {
       return <BaseSidebarScreen ... beforeHideCreateMenu={() => { document.removeEventListener('dragover', this.dragOverListener); }} />
    }
}

```

I hope this helps, do reach out if you have any other questions

Reply:
Thank you so much helps a lot 🙏 . I will try to do as you say. thank you so much.

Reply:
I made a change as you said in the light of the guidance. I hope you will like it @mananjadhav 🙏 👼 . 
```js
if (this.props.beforeHideCreateMenu) {
            this.props.beforeHideCreateMenu();
        }
```
```js
if (this.props.afterShowCreateMenu) {
            this.props.afterShowCreateMenu(this);
        }
```
These conditions may have been extra. I need your opinion as I am not sure. Because I defined it as default props.
Thank you for your help again. I wouldn't dare to change the files 😅.